RUFAS/biophysical/manure/processor.py

```diff
@@ -290,4 +294,90 @@ def _calculate_dissociation_coefficient_of_ammonium(cls, temp: float, pH: float)
             Dissociation coefficient of ammonium (unitless).
 
         """
-        return 1 + 10 ** (0.09018 + 2729.9 / temp - pH)
+        return 1 + 10 ** (0.09018 + 2729.9 / temperature - pH)
+
+    @staticmethod
+    def _determine_outdoor_storage_temperature(air_temperature: float) -> float:
+        """
+        Determines the temperature of the manure in outdoor liquid and slurry storages.
+
+        Parameters
+        ----------
+        air_temperature : float
+            The current day's ambient air temperature (°C).
+
+        Returns
+        -------
+        float
+            The estimated temperature of the manure storage (°C).
+
+        References
+        ----------
+        The temperature bounds of this method were based on personal communication and recommendations from A. Leytem
+        (april.leytem@usda.gov) and A. VanderZaag (andrew.vanderzaag@AGR.GC.CA). These bounds are also support by work
+        from Genedy and Ogejo, 2021 (https://doi.org/10.1016/j.compag.2021.106234) who observed similar minimum and
+        maximum liquid manure temperatures in outdoor clay pit and concrete tank manure storages.
+
+        Notes
+        -----
+        This function clamps stored manure temperature to between 0 and 35 °C. Between 0 and 35 °C, outdoor stored
+        liquid manure temperature is assumed to be equal to ambient air temperature.
+
+        """
+        return float(clip(air_temperature, 0.0, 35.0))
+
+    @staticmethod
+    def determine_barn_temperature(air_temp: float) -> float:
+        """
+        Calculates the barn temperature.
+
+        Parameters
+        ----------
+        air_temp : float
+            Air temperature (c).
+
+        Returns
+        -------
+        float
+            Adjusted barn temperature (c).
+
+        References
+        ----------
+        Between 5 and 30 C, barn temperature is assumed to be equal to outdoor air temperature.
+        This function assumes that barn temperature does not drop below 5 C or increase above 30 C.
+        These bounds were suggested by manure SMEs and are supported by barn temperature ranges
+        reported in Bucklin et al. (FL, upper limit; https://doi.org/10.13031/2013.28851).
+        The lower bound (5 C) suggested by SMEs was based on general industry standards/conditions.
+
+        """
+        return float(clip(air_temp, 5, 30))
```

Comment:
```suggestion
        return float(clip(air_temp, 5.0, 30.0))
```
Functionally I don't think it makes a difference but suggesting this just to be consistent with the other method.